tests/test_wave_05.py

```diff
@@ -80,34 +80,50 @@ def test_create_goal(client):
     }
 
 
-@pytest.mark.skip(reason="test to be completed by student")
+# @pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    raise Exception("Complete test")
+    # raise Exception("Complete test")
     # Act
     # ---- Complete Act Here ----
-
+    response = client.put("/goals/1", json={
+        "title": "Updated Goal Title",
+        "description": "Updated Test Description",
+    })
+    response_body = response.get_json()
     # Assert
     # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert response_body == {
+        "goal": {
+            "id": 1,
+            "title": "Updated Goal Title"
+        }
+    }
+    goal = Goal.query.get(1)
+    assert goal.id == 1
```

Comment:
This test fails with the message:
```python
 goal = Goal.query.get(1)
>       assert goal.id == 1
E       AttributeError: 'Goal' object has no attribute 'id'
```

What do you think is going on here? Have a look at what you named the attribute that refers to the goal's id in your Goal class: https://github.com/Ada-C18/task-list-api/pull/138/files#diff-b2c259b85990b30a9cbf1e8b4dcba2ce11bc2cfb7907527e60f78e1456f201f3R5